client/components/poll-form.html

```diff
@@ -3,7 +3,7 @@
     <form>
         <div class="form-group">
             <label>Nome</label>
-            <input type="text" name="name" class="form-control" placeholder="Nome da questão">
+            <input type="text" name="name" class="form-control" placeholder="Nome da questão" required pattern="[a-zA-Z0-9]+">
```

Comment:
Hi @mrdedavid , thanks for contribution... Could you get the latest version and resolve the conflicts to I able to merge the pattern without spaces?